winrm-elevated.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.extra_rdoc_files = %w[README.md LICENSE]
 
-  s.required_ruby_version = '>= 2.2.0'
+  s.required_ruby_version = '>= 2.3.0'
```

Comment:
I realize 2.2 is ancient but does this gem not work with 2.2?

Reply:
After this PR I can not be sure if the syntax if always ruby 2.2 compatible because rubocop drop the support of ruby one year after the EOL.

https://www.ruby-lang.org/en/news/2018/06/20/support-of-ruby-2-2-has-ended/
https://github.com/rubocop-hq/rubocop/commit/4910d514e12bfa0711d3fbd8a7f9e46e8fb320a4
https://www.jetbrains.com/research/devecosystem-2018/ruby/

Reply:
yeah I think that's reasonable.